nebula_ros/src/continental/continental_ars548_decoder_wrapper.cpp

```diff
@@ -338,10 +360,289 @@ void ContinentalARS548DecoderWrapper::packets_callback(
   }
 }
 
+void ContinentalARS548DecoderWrapper::create_radar_info()
+{
+  namespace continental_ns = nebula::drivers::continental_ars548;
```

Comment:
Since it's still quite a long prefix that's used quite frequently, shortening to `ars548` would also be ok imho.

Reply:
Addressed in https://github.com/tier4/nebula/pull/284/commits/7584e384c5899ebbc93ea9cb8d6749434d19b9a5